docs/applications/wallet.md

```diff
@@ -12,9 +13,22 @@ This wallet is currently only released as a library, and not as a binary.
 This is because it (nor the documentation) is not deemed production
 ready. Use at your own risk, and without too much money depending on it.
 
-## Creating a wallet
+### How is the bitcoin-s wallet implemented
 
-This guide shows how to create a Bitcoin-S wallet and then
+The bitcoin-s wallet is a scalable way for individuals up to large bitcoin exchanges to safely and securely store their bitcoin in a scalable way.
+
+All key interactions are delegated to the [key-manager](key-manager.md) which is a minimal dependecy library to store and use key material.
+
+By default, we store the encrypted root key in `$HOME/.bitcoin-s/encrypted-bitcoin-s-seed.json`. This is the seed that is used for each the wallet on each cryptocurrency network.
+
+The wallet itself is used to manage the utxo life cycle, create transactions, and update wallet balances to show how much money you have the on a cryptocurrency network.
```

Comment:
> on a cryptocurrency network

Since bitcoin-s only supports bitcoin networks, it would make more sense to have this as "on a bitcoin network" and maybe add "(main, testnet, regtest)" for clarity 

applies to above line as well